ui/lib/config-ui/addon/components/messages/page/list.hbs

```diff
@@ -115,4 +115,34 @@
       data-test-action-text="Create message"
     />
   </EmptyState>
+{{/if}}
+
+{{#if this.showMaxMessageModal}}
+  <Hds::Modal
+    id="maximum-message-modal"
+    @size="large"
```

Comment:
This should probably just be the standard modal size (I think you can remove this, because the default is medium)